README.md

```diff
@@ -17,6 +17,104 @@ You can install `SymEngine.jl` by giving the following command.
 julia> Pkg.add("SymEngine")
 ```
 
+## Quick Start
+
+### Working with scalar variables
+
+#### Defining variables 
+One can define variables in a few ways the following 3 examples are equivent.
+Defining 2 variables `a`  and `b` with the names `a` and `b`.
+
+``` julia
+julia> a=symbols(:a); b=symbols(:b)
+b
+
+julia> a,b = symbols("a b")
+(a, b)
+
+julia> @vars a b
+(a, b)
+```
+
+`@vars` is perhaps the most handy for basic use, as it creates a varible whos julia name and whos symbolic name match.
```

Comment:
Can you change this to 
... most handy routine for basic use, as it creates a variable whose `Julia` name and symbolic name match. ?